### DISCUSSION TITLE
Update the readme file with subscriber count and latest video to each channel?

### DISCUSSION THREAD

**Discussion by benthecoder** [benthecoder/yt-channels-DS-AI-ML#7] [Ideas]
Thanks for starring the repo everyone! I was thinking of turning each section into a table, and then add a subscriber count and latest video section, which will update daily using Github Actions and the youtube api. Anyone have any thoughts on this? Any help or suggestion would be appreciated.

**Comment by deveagleistaken** [comment]
that would be cool!

**Comment by Reed1234** [comment]
Helpful!
